manifests/init.pp

```diff
@@ -178,6 +186,8 @@
   Integer[0] $worker_count = min(8, $facts['processors']['count']),
   Boolean $service_enable = true,
   Boolean $service_ensure = true,
+  Integer[0] $content_service_worker_count = (2*min(8, $facts['processors']['count']) + 1),
+  Integer[0] $api_service_worker_count = 1,
```

Comment:
Note to self: this matches the default according to https://docs.gunicorn.org/en/latest/settings.html#workers